src/main/java/prefs/PanelInfo.java

```diff
@@ -1,5 +1,7 @@
 package prefs;
 
+import java.util.Objects;
+
 public class PanelInfo {
```

Comment:
Not a change in this PR, but can you help adding the header comment for this class?